src/u2f/u2f_app.c

```diff
@@ -7,15 +7,37 @@
 
 #include <stddef.h>
 #include <stdio.h>
+#include <wally_bip39.h>
 
 #define APPID_BOGUS_CHROMIUM "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
 #define APPID_BOGUS_FIREFOX "\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"
 
+#define SHORT_MNEMONIC_LENGTH 4
+
+
 typedef struct {
     uint8_t app_id[32];
     const char* name;
 } app_t;
 
+//short menonic for displaying hashes in a more recognizable, way, at the cost of less entropy
+//return number of actually generated words
+static int short_mnemonic(
```

Comment:
we have a convention of prefixing static functions with `_`, i.e. `_short_mnemonic`

Reply:
that's no problem